gsoc.yml

```diff
@@ -1,3 +1,31 @@
+gsoc2018:
+  MananAgarwal:
+    rss_feed: "https://medium.com/feed/@manan_agarwal"
+    project: 'Astropy'
+  sushobhana:
+    rss_feed:  "https://medium.com/feed/@sushobhanapatra"
+    project: 'Astropy'
+  rkiman:
+    rss_feed: "https://rociokiman.blogspot.com/feeds/posts/default?alt=rss" # set up tag? FIXME?
```

Comment:
@rkiman could you update the feed so only gets your GSoC posts?
Read about [how to do so] (https://fedoraproject.org/wiki/How_to_provide_a_category_specific_feed_using_Blogspot_or_Blogger?rd=How_to_provide_a_category_specific_feed_using_Blogger).